siac/daemoncmd.go

```diff
@@ -27,13 +28,36 @@ var (
 		Long:  "Check for available updates.",
 		Run:   wrap(updatecheckcmd),
 	}
+
+	versionCmd = &cobra.Command{
+		Use:   "version",
+		Short: "Print version information",
+		Long:  "Print version information.",
+		Run:   wrap(versioncmd),
+	}
 )
 
 type updateInfo struct {
 	Available bool   `json:"available"`
 	Version   string `json:"version"`
 }
 
+type daemonVersion struct {
+	Version string
+}
+
+// version prints the version of siac and siad.
+func versioncmd() {
+	var versioninfo daemonVersion
+	err := getAPI("/daemon/version", &versioninfo)
+	if err != nil {
+		fmt.Println("Could not get daemon version:", err)
+		return
+	}
+	fmt.Println("Sia Client v" + build.Version)
```

Comment:
If the Sia Daemon is not running, this command will exit before displaying `siac`'s version. Perhaps this line could be moved to the top of the func so as to always print the `siac` version.